src/understanding-deis/concepts.md

```diff
@@ -11,27 +12,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
 methodology and best practices.
 
+## Kubernetes
+
+[Kubernetes][] is an open-source cluster manager developed by Google and
+donated to the [Cloud Native Compute Foundation][cncf]. Kubernetes manages all
+the activity on your cluster. Including converging to a desired state,
+providing critical functions like stable service addresses, health monitoring,
+service discovery and DNS resolution.
+
+Workflow builds upon Kubernetes abstractions like Services, Replication
+Controllers and Pods to provide a developer-friendly UX, source to image, log
+aggregation, etc.
+
+Workflow is shipped as a Kubernetes-native application, installable via
+[Helm][helm]. So operators familiar with Kubernetes will feel right at home
+running Workflow.
+
+For a detailed overview of Workflow components, see our [component][components] break down.
+
 ## Docker
 
-[Docker][] is an open source project to pack, ship and run any
+[Docker][] is an open source project to build, ship and run any
 application as a lightweight, portable, self-sufficient container.
 
-Deis curates your applications as Docker images, which are then
-distributed across your cluster as Docker containers.
+Workflow uses Docker Images and the Docker Engine (via Kubernetes) to package
+and run your application respectively.
 
-(Deis itself is also a set of coordinated Docker containers.)
+If you have not yet converted your application to containers, Workflow provides
+a simple and straightforward "source to Docker image" capability. Supporting
+multiple language runtimes via community [buildpacks][], building your application
+in a container can be as easy as `git push deis master`.
 
 ## Applications
 
-Deis is designed around the concept of an [application][], or app.
-Applications live on a cluster where they use [Containers][]
-to service requests.
+Workflow is designed around the concept of an [application][], or app.
+
+Applications can come in three forms. First, as collection of source files
+stored in a Git repository. Second, as a Dockerfile, which describes how to
+build your app. Third, a reference to an already built Docker Image, hosted on
+a remote repository.
 
```

Comment:
Either comma-delimit or bullet the three forms?  Incomplete sentences read better as bullet points... maybe?